M2/BUILD/mike/Makefile

```diff
@@ -17,6 +17,79 @@ cmake-build: always
 ################
 ## mac os x ####
 ################
+darwin64-gcc8 : always
+	mkdir -p builds.tmp/darwin64-gcc8
+	cd builds.tmp/darwin64-gcc8; ../../../../configure \
+		CC=gcc-8 CXX=g++-8 \
+		--build=x86_64-apple-darwin \
+		--prefix="`pwd`/installed" \
+		--disable-strip \
+		--enable-download
+
+#		--enable-dmg \
+#		CPPFLAGS="-I/usr/local/opt/readline/include" \
+#		LDFLAGS="-L/usr/local/opt/readline/lib" \
+#		--enable-build-libraries="mpir flint pari ntl"
+#		CPPFLAGS="-I/usr/local/include -I/usr/local/opt/readline/include"
+#		LDFLAGS="-L/usr/local/lib -L/usr/local/opt/readline/lib"
+
+#rewrite this: don't change the compiler this way?
+# Thomas Kahle: I rewrote this to use brew's clang latest
+darwin64-latestclang : always
+	mkdir -p builds.tmp/darwin64-latestclang
+	cd builds.tmp/darwin64-latestclang; ../../../../configure \
+		CC="/usr/local/opt/llvm/bin/clang" \
+	        CXX="/usr/local/opt/llvm/bin/clang++" \
+                LDFLAGS="-L/usr/local/opt/llvm/lib -Wl,-rpath,/usr/local/opt/llvm/lib" \
+		OPENMP_CXXFLAGS="-I/usr/local/opt/llvm/include -fopenmp" \
```

Comment:
We don't document any use of variables OPENMP_CXXFLAGS and OPENMP_LDFLAGS.  I bet CXXFLAGS and LDFLAGS would work just as well.  Could you try?